lib/services/services.c

```diff
@@ -241,22 +241,31 @@ resources_action_create(const char *name, const char *standard,
         char *dir = NULL;
         char *buf = NULL;
         struct stat st;
-        for (dir = strtok(strdup(OCF_RA_PATH), ","); dir != NULL; dir = strtok(NULL, ",")) {
+
+        if (dirs == NULL) {
+            services_handle_exec_error(op, ENOENT);
+            return NULL;
```

Comment:
Instead of returning NULL directly, we need to goto return_error so op gets freed